test/array_contraints_test.rb

```diff
@@ -9,4 +9,6 @@
   rejects 123
   rejects [123, 456]
   rejects ['abc', 123]
+  rejects nil
```

Comment:
not sure if we should change it to array, would be solve the array-to-nil issue, but could also be unexpected ... maybe just allowing nil to be valid